src/main/java/de/uni_tuebingen/ub/nppm/db/SaveHelper.java

```diff
@@ -22,52 +25,72 @@ public static void insertMetaData(String form, int id, int benutzerID, int grupp
-                resultRow.put(attribute, value); // Wert zur HashMap hinzufügen
-                results.add(resultRow);
+                for (Object[] row : queryResults) {
+                    Map<String, String> resultRow = new HashMap<>();
+                    resultRow.put(attribute, row[0].toString()); // Wert zur HashMap hinzufügen
+                    results.add(resultRow);
+                }
+            } else {
+                List<String> queryResults = query.getResultList(); // Ergebnisse aus der Datenbank
+                for (String value : queryResults) {
+                    Map<String, String> resultRow = new HashMap<>();
+                    resultRow.put(attribute, value); // Wert zur HashMap hinzufügen
+                    results.add(resultRow);
+                }
             }
-
-            session.getTransaction().commit();
         }
-
         return results;
     }
 
-    public static void updateMetaData(String form, int id, int benutzerID) throws Exception {
-        String sql = "UPDATE " + form + " SET LetzteAenderung = NOW(), LetzteAenderungVon=\"" + benutzerID + "\" WHERE ID=" + id + ";";
-        insertOrUpdate(sql);
-    }
+    public static List<Map> getAttributeMap(String table, String attributeString, int id) throws Exception {
+        List<Map> results = new ArrayList<>();
 
-    public static void updateAttribute(String table, String attribute, String value, int id) throws Exception {
         try ( Session session = getSession()) {
-            session.getTransaction().begin();
-            NativeQuery query = session.createNativeQuery("UPDATE " + table + " SET " + attribute + "= :value WHERE ID= :id");
-            query.setParameter("value", value);
-            query.setParameter("id", id);
-            query.executeUpdate();
-            session.getTransaction().commit();
+
+            String sql = "SELECT " + attributeString + " FROM " + table + " WHERE ID = " + id; // SQL-Abfrage erstellen
+
+            List<Object[]> queryResults = AbstractBase.getListNative(sql); // Ergebnisse aus der Datenbank als Liste von Object-Arrays
+
+            for (Object[] row : queryResults) {
+                Map<String, Object> rowMap = new HashMap<>();
+
+                // Die Spaltenwerte in die Map einfügen, wobei die Spaltennamen als Schlüssel verwendet werden
+                for (int i = 0; i < row.length; i++) {
+                    String columnName = attributeString.split(",")[i].trim(); // Die Spaltennamen sind mit Kommas getrennt
+                    rowMap.put(columnName, row[i]);
+                }
+
+                results.add(rowMap);
+            }
         }
+        return results;
     }
 
-    public static void updateAttribute(String table, String attribute, String value, String id) throws Exception {
-        updateAttribute(table, attribute, value, Integer.parseInt(id));
+    public static List<Map<String, String>> getMapField(String zieltabelle, int id) throws Exception {
```

Comment:
If this is a simple variant of an AbstractBase function, it should be moved into that class.